core/src/main/java/org/pentaho/platform/engine/core/system/objfac/spring/SpringPentahoObjectReference.java

```diff
@@ -34,25 +37,25 @@
  */
 public class SpringPentahoObjectReference<T> implements IPentahoObjectReference<T> {
 
-  private ConfigurableApplicationContext context;
+  private final ConfigurableApplicationContext context;
 
-  private String name;
+  private final String name;
 
   private final Class<T> clazz;
 
-  private IPentahoSession session;
+  private final IPentahoSession session;
 
   private final SpringBeanAttributes attributes;
 
-  private static String PRIORITY = "priority";
+  private static final String PRIORITY = "priority";
 
   public SpringPentahoObjectReference( ConfigurableApplicationContext context, String name, Class<T> clazz,
                                        IPentahoSession session, BeanDefinition beanDef ) {
     this.context = context;
     this.name = name;
     this.clazz = clazz;
     this.session = session;
-    this.attributes = new SpringBeanAttributes( beanDef );
+    this.attributes = new SpringBeanAttributes( beanDef, getContextOwnerPluginId( context ) );
   }
 
   @Override public Class<?> getObjectClass() {
```

Comment:
missed this `@Override public ...` in the same line

Reply:
Addressed!